tests/dataset/test_dataset.py

```diff
@@ -149,7 +149,7 @@ def test_categories():
 
 @pytest.fixture
 def dataset(tmpdir: Path):
-    url = "https://github.com/snuailab/waffle_utils/raw/main/mnist.zip"
+    url = "https://raw.githubusercontent.com/snuailab/assets/main/waffle/sample_dataset/mnist.zip"
```

Comment:
Is it right? I think..."github.com/snuailab/assets/main/waffle/sample_dataset/mnist.zip"

Reply:
https://raw.githubusercontent.com
this url is github special url for raw data.
same same.